ZusExample/Vult/View/AllocationActionStack.swift

```diff
@@ -11,16 +11,24 @@ import PhotosUI
 struct AllocationActionStack: View {
     @EnvironmentObject var vultVM: VultViewModel
     @Environment(\.colorScheme) var colorScheme
+    @State private var isShowingPicker = false
```

Comment:
add var isShowingPicker to VultViewModel

Reply:
Done